.github/workflows/stale.yml

```
@@ -22,3 +22,28 @@ jobs:
           stale-pr-message: |
             This issue has been automatically marked as stale because it has not had recent activity. It will be
             closed if no further activity occurs. Thank you for your contributions.
+  stale1:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-stale: 21
+          stale-issue-message: |
+            This issue will be unassigned soon if no further activity is seen. If the asignees are active please provide any update.
+          include-only-assigned: true
+          remove-stale-when-updated: true
+          only-issue-labels: 'good first issue'  
+
+  stale2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-stale: 28
+          stale-issue-message: |
+            This issue will be labelled 'to be unassigned' in a week because no response has been encountered.
```

At this point, the issue will get the label already, not in week.
```suggestion
            This issue will be labelled 'to be unassigned' because no response has been encountered.
```